src/lib.rs

```diff
@@ -509,3 +520,26 @@ impl DerInner<'_> {
         })
     }
 }
+
+// Format an iterator of u8 into a hex string
+fn hex<'a>(f: &mut fmt::Formatter<'_>, payload: impl IntoIterator<Item = &'a u8>) -> fmt::Result {
+    for b in payload {
```

Comment:
How do we feel about starting this with a `0x` prefix?

Reply:
Added the prefix.